DataFormats/EcalDigi/src/EcalEBPhase2TriggerPrimitiveDigi.cc

```diff
@@ -0,0 +1,69 @@
+#include "DataFormats/EcalDigi/interface/EcalEBPhase2TriggerPrimitiveDigi.h"
+#include <iostream>
+
+EcalEBPhase2TriggerPrimitiveDigi::EcalEBPhase2TriggerPrimitiveDigi() : size_(0), data_(MAXSAMPLES) {}
+
+EcalEBPhase2TriggerPrimitiveDigi::EcalEBPhase2TriggerPrimitiveDigi(const EBDetId& id)
+    : id_(id), size_(0), data_(MAXSAMPLES) {}
+
+void EcalEBPhase2TriggerPrimitiveDigi::setSample(int i, const EcalEBPhase2TriggerPrimitiveSample& sam) {
+  data_[i] = sam;
+}
+
+int EcalEBPhase2TriggerPrimitiveDigi::sampleOfInterest() const {
+  if (size_ == 1)
```

Comment:
I know what is going on and I will be working on this for quite some time yet, during which  maybe this will change

Reply:
When this code gets integrated into the official CMSSW release also other people will look at it and need to understand it. They should not need to come to you and ask what it does.
From the code alone one can not deduce that this is still being worked on.
If the code may change in the future this can be also mentioned in the comment.